inlong-manager/manager-web/sql/changes-1.8.0.sql

```diff
@@ -25,3 +25,22 @@ USE `apache_inlong_manager`;
 
 ALTER TABLE inlong_group
     CHANGE lightweight inlong_group_mode tinyint(1) DEFAULT 0 NULL COMMENT 'Inlong group mode, Standard mode: 0, DataSync mode: 1';
+
```

Comment:
It's better to add more annotations to show the purpose of this change. Like `For support multi-tenant management in InLong, see https://github.com/apache/inlong/pull/8090`.

Reply:
fixed, thx.